tox.ini

```diff
@@ -39,7 +39,7 @@ whitelist_externals=make
 basepython=python
 extras=lint
 commands=
-    mypy -p {toxinidir}/eth_typing --config-file {toxinidir}/mypy.ini
```

Comment:
Where did you see a failure before this change? Just curious when it triggers. It's configured like this in the project template, so if that's wrong, then we should upstream the change.

Reply:
`mypy` got mad about there being a slash in the module name.